src/globals.c

```diff
@@ -711,22 +711,22 @@ void global_print_help(void)
 {
 	print("Global Options:\n");
 	print("   -h, --help              Show help options\n");
-	print("   -p, --path=[PATH...]    Use [PATH...] as the path to verify (eg: a chroot or btrfs subvol)\n");
+	print("   -p, --path=[PATH]       Use [PATH] as the path to verify (eg: a chroot or btrfs subvol)\n");
 	print("   -u, --url=[URL]         RFC-3986 encoded url for version string and content file downloads\n");
 	print("   -P, --port=[port #]     Port number to connect to at the url for version string and content file downloads\n");
 	print("   -c, --contenturl=[URL]  RFC-3986 encoded url for content file downloads\n");
 	print("   -v, --versionurl=[URL]  RFC-3986 encoded url for version file downloads\n");
 	print("   -F, --format=[staging,1,2,etc.]  the format suffix for version file downloads\n");
+	print("   -S, --statedir=[PATH]   Specify alternate swupd state directory\n");
+	print("   -C, --certpath=[PATH]   Specify alternate path to swupd certificates\n");
+	print("   -W, --max-parallel-downloads=[n] Set the maximum number of parallel downloads\n");
```

Comment:
Same here, the manpage also has that problem (the option is listed but the fact it needs a value is not explicit).

There is also a discrepancy between this help and the manpage, the manpage indicates the short version is `-D` which is deprecated so it would be good to update it.